pkg/reconciler/finalizer.go

```diff
@@ -38,6 +38,8 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, pr *tektonv1.PipelineRun)
 			return err
 		}
 
+		logger = logger.With("Namespace", pr.Namespace)
+		r.eventEmitter.SetLogger(logger)
```

Comment:
eventEmitter is a shared field 
adding ns in logger will affect all reconciliation
you can update in EmitMessage rather than here if required.

Reply:
Ya right
Let me re check on this